docs/scss/_community.scss

```diff
@@ -0,0 +1,6 @@
+.ov-community {
```

Comment:
this class is probably best suited for a wrapper since it's not more specific.

Reply:
That makes sense to me. It should probably just be used for overall page styling then? For more specific community classes it could be named `.ov-community-section` or something along those lines.